contrib/checkstyle/test/src/mill/contrib/checkstyle/CheckstyleModuleTest.scala

```diff
@@ -0,0 +1,154 @@
+package mill
+package contrib.checkstyle
+
+import mill.api.Result
+import mill.scalalib.{JavaModule, ScalaModule}
+import mill.testkit.UnitTester
+import mill.testkit.TestBaseModule
+import utest._
+
+object CheckstyleModuleTest extends TestSuite {
+
+  val resources = os.Path(sys.env("MILL_TEST_RESOURCE_FOLDER"))
+
+  def tests = Tests {
+
+    test("checkstyle") {
+
+      test("format") {
+
+        test("plain") {
+          assert(
+            cs("plain").checkJava(resources / "compatible-java"),
+            cs("plain").checkJava(
+              resources / "sbt-checkstyle",
+              "Utility classes should not have a public or default constructor"
```

Comment:
Is it possible to add a few more things for checkstyle to check for, and assert that they get found? Just to exercise a few more code paths and make sure it works for more than just a single error